### PR TITLE
General: Update changelog for internal point release

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* This PR updates the readme with the most recent changes made for the 9.5.2 release.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No.
#### Testing instructions:

* Check for typos.

### CODE REVIEW

**Review by leogermani — APPROVED** [review]
LGTM

**Review by fgiannar — APPROVED** [review]
LGTM 👍

**Comment by jeherve** [comment]
Sorry both, I had forgotten that I needed a changelog entry for my changelog update. :) This will now need another review.

**Comment by anomiex** [comment]
Shouldn't CHANGELOG.md also be updated here?

**Comment by jeherve** [comment]
> Shouldn't CHANGELOG.md also be updated here?

You're right. Done in c750434ed0dcbe4bff9e4e3baeec2fc6e8e9b9b5

File: projects/plugins/jetpack/changelog/update-changelog-952 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: compat
+Comment: Changelog: this is the changelog. :)
```
Comment (anomiex) [inline projects/plugins/jetpack/changelog/update-changelog-952:3]: Heh. It does ignore CHANGELOG.md when deciding whether to require a change file, but not readme.txt.

**Review by anomiex — APPROVED** [review]

**Pull request merged**